src/renderer/contexts/KeybindingsContext.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { createContext, useEffect } from 'react'
+
+import {
+  ActionEmitter,
+  KeybindAction,
+  keyDownEvent2Action,
+} from '../keybindings'
+import { useKeyBindingAction } from '../hooks/useKeyBindingAction'
+import { useDialog } from '../hooks/useDialog'
+import Settings from '../components/dialogs/Settings'
+import { Screens } from '../ScreenController'
+import KeybindingCheatSheet from '../components/dialogs/KeybindingCheatSheet'
+
+import type { PropsWithChildren } from 'react'
+
+export const KeybindingsContext = createContext(null)
+
+export const KeybindingsContextProvider = ({
+  children,
+}: PropsWithChildren<{}>) => {
+  const { openDialog } = useDialog()
+
+  // @TODO: This probably needs another place
+  useKeyBindingAction(KeybindAction.Settings_Open, () => {
+    if (window.__screen === Screens.Main) {
+      // Only if user is logged in & open settings if not already opened
+      if (!window.__settingsOpened) {
+        openDialog(Settings)
+      }
+    }
+  })
+
+  // @TODO: This probably needs another place
+  useKeyBindingAction(KeybindAction.KeybindingCheatSheet_Open, () => {
+    if (!window.__keybindingsDialogOpened) {
+      openDialog(KeybindingCheatSheet)
+    }
+  })
```

Comment:
These key-bindings here are important on a base-level for the application but still they don't belong here (as this context should be an agnostic of how it is used).

I will probably move them somewhere else